.github/workflows/publication_maven.yml

```diff
@@ -0,0 +1,59 @@
+name: Publication to MavenCentral
+
+on:
+  workflow_dispatch:
+    inputs:
+      stableVersion:
+        required: true
+        type: string
+        description: 'Stable version'
+      snapshotVersion:
+        required: true
+        type: string
+        description: 'Snapshot version'
+
+jobs:
+  publication:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Set up environment variables
+        env:
+          OSSRH_USER: ${{ secrets.ossrh_user }}
+          OSSRH_PASSWORD: ${{ secrets.ossrh_password }}
+          OSSRH_STAGING_PROFILE_ID: ${{ secrets.ossrh_staging_profile_id }}
+          PGP_KEYID: ${{ secrets.pgp_keyid }}
+          PGP_KEY: ${{ secrets.pgp_key }}
+          PGP_PASSWORD: ${{ secrets.pgp_password }}
+          STABLE_VERSION: ${{ github.event.inputs.stableVersion }}
```

Comment:
Project's [gradle.properties](https://github.com/KasperskyLab/Kaspresso/blob/c83189c06594eaea3fdf9e239a2ff33c740ea553/gradle.properties#L9)  should be the source of truth. Why do you want to override it here?

Reply:
It seems that local.properties will be a better place

Reply:
Remove version input